testing_utils_test.go

```diff
@@ -40,9 +40,14 @@ const (
 	fakeAppNameBroken = "/fake-app-broken"
 	fakeDeploymentID  = "867ed450-f6a8-4d33-9b0e-e11c5513990b"
 	fakeAPIFilename   = "./tests/rest-api/methods.yml"
-	fakeAPIPort       = 3000
 )
 
+var (
+	fakeResponses map[string]string
+	once          sync.Once
+)
+
+// restMethod represents a fake response
```

Comment:
maybe better: `restMethod represents an expected HTTP method and an associated fake response`